pkg/metadataapi/server/api/status.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	utilapi "github.com/puppetlabs/leg/httputil/api"
+	"github.com/puppetlabs/relay-core/pkg/metadataapi/errors"
+	"github.com/puppetlabs/relay-core/pkg/metadataapi/server/middleware"
+	"github.com/puppetlabs/relay-core/pkg/model"
+)
+
+type GetActionStatusResponseEnvelope struct {
+	ExitCode int    `json:"exit_code"`
+	Status   string `json:"status"`
```

Comment:
Think this is unused?

Reply:
Ahh, nice catch. I stripped out the "status" fields for now (but missed this one), as I want to be very deliberate about how/where the status gets set, and I haven't worked up that design yet.